crates/astria-composer/src/executor/tests.rs

```diff
@@ -112,15 +115,34 @@ async fn setup() -> (MockServer, MockGuard, Config, NamedTempFile) {
         grpc_addr: "127.0.0.1:0".parse().unwrap(),
         fee_asset: "nria".parse().unwrap(),
     };
-    (server, startup_guard, cfg, keyfile)
+    (server, cfg, keyfile)
+}
+
+/// Assert that given error is of correct type and contains the expected chain IDs.
+fn assert_chain_id_err(
```

Comment:
To get a better error diagnostic:
```suggestion
#[track_caller]
fn assert_chain_id_err(
```

Reply:
Done in 33ae83d.